app/lib/shared/utils.dart

```diff
@@ -417,3 +417,12 @@ String createUuid([List<int> bytes]) {
     bytes.sublist(10).map(formatByte).join(),
   ].join('-');
 }
+
+bool isSemanticVersion(String version) {
+  try {
+    semver.Version.parse(version);
+    return true;
+  } catch (_) {
```

Comment:
Can this be anything other than `FormatException`?

Reply:
Just saying, if we want to be perfectly safe, it's probably better to ignore `FormatException`, and if throws anything log it with `shout()` (and ignore it).. Or just only catch `FormatException` I'm pretty sure that's the only thing this will throw.